h/traversal/__init__.py

```diff
@@ -77,4 +81,6 @@
     "UserByIDRoot",
     "UserRoot",
     "GroupContext",
+    "GroupMembershipContext",
+    "group_membership_api_factory",
```

Comment:
Should really tidy up thie `__init__.py` file, there's a bunch of stuff in it that doesn't need to be, and the `__all__` is unnecessary.